deployments/kubehound/embed.go

```diff
@@ -0,0 +1,12 @@
+package embedconfigdocker
+
+import (
+	"embed"
+)
+
+const (
+	DefaultComposePath = "docker-compose.embed.yaml"
+)
+
+//go:embed docker-compose.embed.yaml
+var F embed.FS
```

Comment:
Just a note, as I'm not sure exactly how you want to handle this but:
you can use `var F string` directly, and avoid the need to readfile / do the error checks  / convert to byte slice/string.

This also removes the need for the `DefaultComposePath`.

(if you want to provide another file, dynamically, you'll also need to change the FS, but that can be done on the caller  or here, depends)